packages/benchmarks/src/radiantModels.test.ts

```diff
@@ -0,0 +1,34 @@
+/**
+  @description Quick test to make sure all the Radiant models are functional.
+  Useful to test before executing benchmark runs to ensure all models are working.
+ */
+import { assertEnvVars } from "mongodb-rag-core";
+import { envVars } from "./envVars";
+import { makeRadiantChatLlm } from "./makeRadiantChatLlm";
+import { radiantModels } from "./radiantModels";
+
+jest.setTimeout(60000);
+// NOTE: due to this issue https://github.com/nodejs/node/issues/39964,
+// you must run the tests with a Node version >= 20.0.0
+describe.skip("Radiant models", () => {
```

Comment:
Test won't do us much good if we're skipping it. Can we track a follow up for this as an EAI?

Reply:
i dont think we need to follow up b/c we only run the benchmarks from our local envs, so don't need to make sure it works from the cloud. this is just to smoke test that everything is functional w/ radiant